content/app/github/wrappers/file-element-wrapper.js

```diff
@@ -37,6 +39,18 @@ define("content/app/github/wrappers/file-element-wrapper", function(require, exp
 		 */
 		constructor : function (domElement) {
 			FileElementWrapper.$super.call(this, domElement);
+			this._fileName = null;
+			this._init();
+		},
+
+		/**
+		 * File name.
+		 * @type {String}
+		 */
+		fileName : {
+			get : function() {
+				return this._fileName;
+			}
 		},
```

Comment:
Interesting, maybe I should remove file-data-element-wrapper.js and move code for getting lines here...